StackClassImplementation/StackClassImplementation/Program.cs

```diff
@@ -6,7 +6,97 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Stack Implementation Program");
+            Console.WriteLine("Choose what you want\n1-integer Stack \n2-Employee Stack");
+            int ch;
+            string input = Console.ReadLine();
+            if (int.TryParse(input, out ch))
+            {
+                CallingProperMethod(ch);
+            }
+            else
+            {
+                Console.WriteLine("Please enter numbers only");
+            }
+
+        }
+
+        private static void CallingProperMethod(int ch)
+        {
+            switch (ch)
+            {
+                case 1:
+                    IntegerStack();
+                    break;
+                case 2:
+                    EmployeeStack();
+                    break;
+                default:
+                    Console.WriteLine("Please enter 1 or 2");
+                    break;
+            }
         }
+
+        private static void IntegerStack()
+        {
+
+            Console.WriteLine("Creating Integer Stack...");
+            Stack<int> intStack = new Stack<int>();
+            Console.WriteLine("Filling Stack with 20 integer values");
+            FillIntStack(intStack);
+            intStack.Print();
+            Console.WriteLine("Peak item: " + intStack.Peak());
+            Console.WriteLine($"Popped 2 items...({intStack.Pop()} & {intStack.Pop()})");
+            Console.WriteLine("Our Stack now contains " + intStack.Count + " items");
+            intStack.Print();
+            Console.WriteLine("Clearing Stack...");
+            intStack.Clear();
+            Console.WriteLine("Our Stack now contains " + intStack.Count + " items");
+            intStack.Print();
+        }
+
+        private static void EmployeeStack()
+        {
+
+            Console.WriteLine("Creating Employees Stack...");
+            Stack<Employee> EmpStack = new Stack<Employee>();
+            Console.WriteLine("Filling Stack with 20 Employees ");
+            FillEmpStack(EmpStack);
+            EmpStack.Print();
+            Console.WriteLine("Peak item: " + EmpStack.Peak());
+            Console.WriteLine($"Popped 2 items...({EmpStack.Pop()} & {EmpStack.Pop()})");
+            Console.WriteLine("Our Stack now contains " + EmpStack.Count + " items");
+            EmpStack.Print();
+            Console.WriteLine("Clearing Stack...");
+            EmpStack.Clear();
+            Console.WriteLine("Our Stack now contains " + EmpStack.Count + " items");
+            EmpStack.Print();
+        }
+
+        private static void FillEmpStack(Stack<Employee> empStack)
+        {
+            String[] fname = { "Liam ", "David ", "Henry","Michael", "Oliver ", "James ",
```

Comment:
try to use more `var` instead of specifying the types explicitly. Try to update this in this file and the others